MonoGame.Framework/Input/GamePad.cs

```diff
@@ -107,5 +107,16 @@ public static bool SetVibration(int index, float leftMotor, float rightMotor)
             
             return PlatformSetVibration(index, MathHelper.Clamp(leftMotor, 0.0f, 1.0f), MathHelper.Clamp(rightMotor, 0.0f, 1.0f));
         }
+
+        /// <summary>
+        /// Returns the maximum number of game pads supported on this system.  Attempting to
+        /// access a gamepad index higher than this number will result in an <see cref="InvalidOperationException"/>
+        /// being thrown by the API.
+        /// </summary>
+        /// <returns>The maximum number of game pads supported on this system.</returns>
+        public static int GetMaximumGamePadIndex()
+        {
+            return PlatformGetMaxNumberOfGamePads();
+        }
```

Comment:
This seems to confuse what the difference in a count and an index is.  Let's just make it clear:
```
public static int MaximumGamePadCount { get; }
```
This is now clearly a count of gamepads like one would expect.